src/pages/index.tsx

```diff
@@ -36,50 +35,73 @@ export default function Home(): JSX.Element {
       <main>
         <section className={styles.features}>
           <div className="container">
-            <div className="row">
-              <Grid cols={3} gap="2rem">
-                <Card
-                  title="Set Up Your Accounts"
-                  href="/docs/guides/build-it-yourself/landing-zone/intro/what-youll-learn-in-this-guide"
-                >
-                  Streamline how you create, configure, and secure AWS accounts
-                  and multi-account structures.
-                </Card>
-                <Card
-                  title="Configure a CI/CD Pipeline"
-                  href="/docs/guides/build-it-yourself/pipelines/intro/what-youll-learn-in-this-guide"
-                >
-                  Use your preferred CI tool to set up an end‑to‑end pipeline
-                  for your infrastructure code.
-                </Card>
-                <Card
-                  title="Achieve Compliance"
-                  href="/docs/guides/build-it-yourself/achieve-compliance/intro/what-youll-learn-in-this-guide"
-                >
-                  Implement the CIS AWS Foundations Benchmark using our curated
-                  collection of modules and services.
-                </Card>
-                <Card
-                  title="The Reference Architecture"
-                  href="/docs/guides/reference-architecture/overview/overview"
-                >
-                  Bootstrap your infrastructure in about a day by letting
-                  Gruntwork deploy a Reference Architecture customized just for
-                  you.
-                </Card>
-                <Card
-                  title="Deploy a Service"
-                  href="/docs/guides/build-it-yourself/overview"
-                >
-                  Learn how to deploy Gruntwork services to construct your own
-                  bespoke architecture.
-                </Card>
-                <Card title="Courses" href="/docs/courses">
-                  Learn DevOps fundamentals with our series of introductory
-                  video tutorials.
-                </Card>
-              </Grid>
-            </div>
+            <CardGroup>
+              <Card
+                title="What is Gruntwork?"
+                href="/docs/intro/overview/intro-to-gruntwork"
+                icon="/img/icons/learn.svg"
+              >
+                Learn how Gruntwork products can help you deploy a world class
+                infrastructure.
+              </Card>
+              <Card
+                title="The Reference Architecture"
+                href="/docs/guides/reference-architecture/overview/overview"
+                icon="/img/icons/refarch.svg"
+              >
+                Bootstrap your infrastructure in about a day by letting
```

Comment:
Nit: This feels more like a marketing blurb and less like verbiage that will help customers find docs for interacting with their Ref Arch.

I could see something like: "Got a Ref Arch deployed? Learn all about it" <-- not this exact text...but the gist should be "if you got the ref arch, this is where you can learn about how to absorb it into your org"

Reply:
Yeah, that's fair. Updated!